spot-contracts/contracts/FeePolicy.sol

```diff
@@ -400,12 +417,6 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
             );
         }
 
-        // We skip "dust" value transfer
-        uint256 minPerpAbsValueDelta = s.perpTVL.mulDiv(EQUILIBRIUM_REBALANCE_PERC, ONE);
```

Comment:
Is it possible for this check to still be relevant, even when there's an equilibrium range?

For example, DR is out of range, but also the amount of liquid AMPL in the vault is very low. So a rebalance would still trigger on a small amount.

Reply:
Well it can't though if the dr range is wide enough.. A dr range of [0.98, 1.02], would mean ~2% perp tvl perc delta?

Reply:
Say the dr equilibrium range is [0.98, 1.02], and the DR is at 0.5. There's 250,001 liquid AMPL in the rotation vault, and it reserves 250K AMPL that it doesn't use to rotate. That means the DR is out of range, but the rotation size would be 1 AMPL. This could be considered dust

Reply:
ah. Good point .. yes

Reply:
The liquidity thresholds just limit rollovers though. This is related to rebalancing?